ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -326,18 +326,6 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ovn
-- as: azure-ipi-private-workers-marketimage-p2-f14
```

Comment:
I guess we need to replace it with `azure-ipi-private`

Reply:
yes, will update